src/olympia/versions/admin.py

```diff
@@ -24,6 +24,8 @@ class Media:
 
     view_on_site = False
     readonly_fields = ('id', 'created', 'version', 'channel')
+    list_display = ('id', 'addon_guid', 'version', 'deleted', 'channel')
+    list_select_related = ('addon',)
```

Comment:
I'm only displaying the add-on guid and not the add-on str representation/name so it shouldn't cause extra queries, and that's also why I didn't bother going with the `prefetch_related` route, since a join is enough.

I verified manually that it doesn't cause extra queries but I should add a test.

Reply:
Ugh I totally messed up, mixed together `VersionAdmin` (which I changed for convenience when testing) and `InstallOriginAdmin`. Will fix and add tests.